src/containers/forgotPasswordReset/index.tsx

```diff
@@ -13,7 +13,7 @@ interface NewPasswords {
 }
 
 const ForgotPasswordReset: React.FC = () => {
-  const { key } = useParams();
+  const { key } = useParams<{ key: string }>();
```

Comment:
This is good enough, but if we wanted to be consistent with how we do types in other places of the code, or if there is a possibility that the # of params could increase, we could define an interface/type and pass that too.

`
type ForgotPasswordParams {
    key: string;
}
const { key } = useParams<ForgotPasswordParams>();
`
Totally optional, but otherwise looks good!